frontend/src/assets/locales/ro/translation.json

```diff
@@ -48,5 +49,10 @@
   "pagination": {
     "rows_per_page": "Randuri pe pagina",
     "range_separator_text": "din"
+  },
+  "access_codes": {
+    "create": "Creeaza cod nou",
+    "availability": "Valabilitate",
```

Comment:
Will those texts be used in other places? I haven't seen any in the design, and because of that i don't think they should be in general. Shouldn't general only have global translations used in multiple places?